test/integration/services/modal.js

```diff
@@ -194,7 +194,7 @@ context('Modal Service', function() {
       .contains('Custom Footer');
 
     cy
-      .get('.fill-window--dark')
+      .get('.fill-window--dark.is-shown')
```

Comment:
These were just grabbing the wrong element, since we changed the class name for these to have `.is-shown`.

Reply:
seems odd that this is needed.. was `get` getting more than one or something?

Reply:
Ya, the `get` is returning other `<div class="fill-window--dark">` elements, that don't have `.is-shown`.